trillian/ctfe/handlers.go

```diff
@@ -219,6 +219,25 @@ type CertValidationOpts struct {
 	extKeyUsages []x509.ExtKeyUsage
 }
 
+// NewCertValidationOpts can be used for struct creation outside of ctfe package.
+func NewCertValidationOpts(trustedRoots *PEMCertPool, currentTime time.Time, rejectExpired bool, rejectUnexpired bool, notAfterStart *time.Time, notAfterLimit *time.Time, acceptOnlyCA bool, extKeyUsages []x509.ExtKeyUsage) CertValidationOpts {
+	var vOpts CertValidationOpts
+	vOpts.trustedRoots = trustedRoots
+	vOpts.currentTime = currentTime
+	vOpts.rejectExpired = rejectExpired
+	vOpts.rejectUnexpired = rejectUnexpired
+	vOpts.notAfterStart = notAfterStart
+	vOpts.notAfterLimit = notAfterLimit
+	vOpts.acceptOnlyCA = acceptOnlyCA
```

Comment:
(style) probably neater to initialize inline:
```golang
	vOpts := CertValidationOpts{
		trustedRoots: trustedRoots,
		currentTime: currentTime,
		rejectExpired: rejectExpired,
		rejectUnexpired: rejectUnexpired,
		notAfterStart: notAfterStart,
		notAfterLimit: notAfterLimit,
		acceptOnlyCA: acceptOnlyCA,
	}
```

Reply:
Done.